articles/azure/azs-how-link-storage-explorer.md

```diff
@@ -24,6 +24,10 @@ In this article, you will learn how to link Azure Stack Hub subscriptions to the
 
 If you need to install Azure Storage Explorer, please click [here](https://azure.microsoft.com/en-us/features/storage-explorer/) and install it.
 
+## Official documentation
```

Comment:
Maybe change this to Microsoft documentation (our doc is official too :)

Reply:
What about Official Microsoft documentation?

Reply:
Ok that makes sense to me.  I'll go through the other Azure docs now and make sure it's consistent.  So as a general practice, we were thinking of calling these sections "{vendor_name} Documentation" (e.g. Microsoft Documentation) unless it's a mixture of vendors which in that case we'll just say "Vendor Documentation".  Would that be the best method?

Reply:
Sounds good. Remember it's a lowercase "d" for documentation though so headings have consistent capitalisation :)

Reply:
Fixed in commit 013b730